presentation/views/survey_export.slim

```diff
@@ -94,7 +94,7 @@ javascript:
     
     // Append submit btn to last page
     if(i == #{{pages}}.length) {
-      button_str = "<button type='submit' class='btn btn-primary float-right mb-4' id='btn-submit'>Submit</button>"
+      button_str = "<button type='submit' class='btn btn-primary float-right mb-4' id='btn-submit'>Submit (提交)</button>"
```

Comment:
I think one major todo for future might be better internationalization (i18n) so that users can switch languages. Right now, many/most of our surveys are done in N. America and they might be confused to see Chinese words. Similarly, Taiwanese might be nervous to see English words!  Let's discuss.  But for now, can we put a comment above the lines that use Chinese characters so we can find them easily later? E.g.:
```javascript
// ZH used here
```

Reply:
note there are other files that introduce Chinese characters – please apply suggested comment to them too!